src/utils/paymentQRHelper.ts

```diff
@@ -59,9 +61,14 @@ export const getUpdatedTransactionsPaymentQRIdentifiers = (
       if (paymentQRPayload) {
         // TODO: Expand support for payment QRs
         // TODO: Perform preliminary parsing to determine payload type, then perform parsing
-        const paymentQRRecords = parseAndValidateSGQR(
-          paymentQRPayload
-        ) as Record<string, unknown>;
+        const paymentQR = parseAndValidateSGQR(paymentQRPayload) as PaymentQR;
```

Comment:
parseAndValidateSGQR() return type as `SGQR`..can explain why we assign as `PaymentQR`. 

and 

how about the 2nd parameter of `metaProperties`

Reply:
The end goal is to use `parsePaymentQR` in `@rationally-app/payment-qr-parser` which parses all payment QRs!

The return type for `parsePaymentQR` is `PaymentQR`, which contains all the possible properties that a parsed payment QR can contain. A later PR will change the use of `parseAndValidateSGQR` to `parsePaymentQR`.

The current version of `@rationally-app/payment-qr-parser` is `1.2.2` as of this PR, `metaProperties` does not exist then. This will be replaced with `parsePaymentQR` as well in a later PR, which will include `metaProperties`.

Reply:
thanks @yongjiajie  for explaining 🙏